From cd85b05864cb79edcb88b691629b0a012367c235 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Tue, 18 Feb 2020 10:52:18 +0800
Subject: [PATCH] ARM: rockchip_defconfig: remove
 CONFIG_CPU_FREQ_GOV_PERFORMANCE=y

CONFIG_CPU_FREQ_GOV_PERFORMANCE is auto selected by
CONFIG_CPU_FREQ_DEFAULT_GOV_INTERACTIVE.

Change-Id: Ic5315e7fe80da246761a7da1628a7e705cb8aef4
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 787876c5e675..7b7235d496bd 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -56,7 +56,6 @@ CONFIG_CPU_FREQ=y
 CONFIG_CPU_FREQ_STAT=y
 CONFIG_CPU_FREQ_TIMES=y
 CONFIG_CPU_FREQ_DEFAULT_GOV_INTERACTIVE=y
-CONFIG_CPU_FREQ_GOV_PERFORMANCE=y
 CONFIG_CPU_FREQ_GOV_POWERSAVE=y
 CONFIG_CPU_FREQ_GOV_USERSPACE=y
 CONFIG_CPU_FREQ_GOV_ONDEMAND=y
-- 
2.35.3

